### PR TITLE
Code clear up - RelPath + commented code deleting

### PR DESCRIPTION
**Pull request by variableundefined**
This clears up some relative pathing & a few snippets of unused, commented out code.

I decided to stop here and leave the rest for another PR or three since this PR is getting a bit too large and unwieldly at this point.

🆑:
fix: Some relative path & commented out code has been cleared up in code. Report unexpected issues.
/🆑

### CODE REVIEW

File: code/game/objects/effects/spawners/gibspawner.dm [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
-		gibamounts = list(1,1,1,1,1,1,1)
+/obj/effect/gibspawner/human
+	gibtypes = list(/obj/effect/decal/cleanable/blood/gibs,/obj/effect/decal/cleanable/blood/gibs/down,/obj/effect/decal/cleanable/blood/gibs,/obj/effect/decal/cleanable/blood/gibs,/obj/effect/decal/cleanable/blood/gibs,/obj/effect/decal/cleanable/blood/gibs,/obj/effect/decal/cleanable/blood/gibs/core)
+	gibamounts = list(1,1,1,1,1,1,1)
 
-		New()
-			gibdirections = list(list(NORTH, NORTHEAST, NORTHWEST),list(SOUTH, SOUTHEAST, SOUTHWEST),list(WEST, NORTHWEST, SOUTHWEST),list(EAST, NORTHEAST, SOUTHEAST), alldirs, alldirs, list())
-			gibamounts[6] = pick(0,1,2)
-			..()
+/obj/effect/gibspawner/New()
```
Comment (variableundefined) [inline code/game/objects/effects/spawners/gibspawner.dm]: woops

File: code/__HELPERS/icons.dm [hunk 2]
```diff
@@ -215,81 +215,80 @@ world
 
 #define TO_HEX_DIGIT(n) ascii2text((n&15) + ((n&15)<10 ? 48 : 87))
 
-icon
-	proc/MakeLying()
-		var/icon/I = new(src,dir=SOUTH)
-		I.BecomeLying()
-		return I
+icon/proc/MakeLying()
```
Comment (Fox-McCloud) [inline code/__HELPERS/icons.dm]: `/icon`

**Comment by SamHPurp** [comment]
Good god what a monster. The sooner this is tested and merged the better.

**Comment by tigercat2000** [comment]
I don't see any issues with this at an overview. I think we should test-merge this for a few days to make sure nothing obvious breaks, then just bite the bullet.

It's worth nothing that you should toy with your editor settings and disable the behavior of automatically adding a newline to files whenever they're saved, because I see that on some of these files.

**Comment by variableundefined** [comment]
I've changed editor since this PR was made to avoid that behavior

**Pull request merged**